modules/vpc/main.tf

```diff
@@ -39,3 +40,27 @@ module "vpc" {
     "kubernetes.io/cluster/${var.kubernetes_cluster_name}" = "shared"
   }
 }
+
+module "nat_instance" {
+  # create nat instance instead of nat gateway
+  count  = var.enable_nat_gateway ? 0 : 1
+  source = "int128/nat-instance/aws"
```

Comment:
Please pin a version for the module. Actually, for some reason the VPC module isn't pinned either. Mind adding one there as well?

Reply:
version: for nat module: 2.0.0, for VPC module: 2.66.0